Builder/Builder/Program.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+namespace Builder
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var director = new Director();
+            var whitePizzabuilder = new WhitePizzaBuilder();
```

Comment:
It would be better if you used polymorphism to define builders, without creating new builder variable for each one.